Source/FluentSpreadsheets/Sheets/SegmentImplementations/PrototypeHeaderRowFooterSegmentBase.cs

```diff
@@ -0,0 +1,19 @@
+using FluentSpreadsheets.Sheets.Segments;
+using FluentSpreadsheets.Sheets.Visitors;
+
+namespace FluentSpreadsheets.Sheets;
+
+public abstract class PrototypeHeaderRowFooterSegmentBase<THeaderSource, THeaderDestination, TRowData, TFooterData> :
```

Comment:
Maybe simplify it to PrototypeSegmentBase? And HeaderRowFooterSegmentBase to SegmentBase

Reply:
Yeah, I think it'll work better that way